presto-main/src/main/java/com/facebook/presto/sql/ExpressionUtils.java

```diff
@@ -264,4 +264,14 @@ public Expression rewriteQualifiedNameReference(QualifiedNameReference node, Voi
             }
         }, expression);
     }
+
+    public static Expression andOrTrue(List<Expression> expressions)
```

Comment:
There's a `combineConjuncts` method in this class that does this (and a bit more), already.